core/src/main/scala/org/apache/spark/ui/jobs/StagePage.scala

```diff
@@ -447,6 +464,22 @@ private[ui] class StagePage(parent: StagesTab, store: AppStatusStore) extends We
     </script>
   }
 
+  def exceptionSummaryTable(exceptionSummary: Seq[ExceptionSummary]): Seq[Node] = {
+    val propertyHeader = Seq("Exception", "Message", "Count", "Sample Details")
```

Comment:
Nit: `Sample Details` => `Sample`

Reply:
The reason I use sample is there could be exception which has same message but different stacktrace, but I just show one stacktrace here. I'll change it into details.